frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -48,6 +48,10 @@ export interface OperatorValueSelectProps {
      * i.e. it limits the options shown from the options that would have been shown
      * **/
     operatorAllowlist?: Array<PropertyOperator>
+    /**
+     * Force single-select mode regardless of operator type
+     * **/
+    forceSingleSelect?: boolean
```

Comment:
Why do we need to force a single select here? Couldn't someone conceivably have a quick filter e.g. Europe that includes many values